src/app/containers/MostRead/utilities/index.js

```diff
@@ -5,8 +5,11 @@ const SHOULD_RENDER_LAST_UPDATED_TIME = 60 * 24 * 60 * 60 * 1000; // 60 days
  * @param {string} formattedTimestamp A formatted timestamp (e.g. 2019-11-06T16:49:00Z )
  */
 export const mostReadRecordIsFresh = formattedTimestamp => {
+  // Note in storybook this value will be 'Friday, 9 August 2019 14:04:14 GMT'
+  // Because of timemachine (simorgh/.storybook/time-machine.js)
   const now = Date.now();
-  const utcTimestamp = Date.parse(formattedTimestamp);
```

Comment:
I changed this to new Date() because on storybook i was getting an error with Date.parse in storybook. 
![image](https://user-images.githubusercontent.com/30599794/71719055-074e8a00-2e15-11ea-9831-94669ead2d9f.png)